core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/ConfigDocItemFinder.java

```diff
@@ -496,7 +497,7 @@ private List<String> extractEnumValues(TypeMirror realTypeMirror, boolean useHyp
                     // Show enum constant description as a Tooltip
                     String javaDoc = enumJavaDocParser.parseConfigDescription(rawJavaDoc);
                     acceptedValues.add(String.format(Constants.TOOLTIP, enumValue,
-                            javaDoc.replace("<p>", Constants.EMPTY).replace("</p>", Constants.EMPTY)));
+                            javaDoc.replace("<p>", EMPTY).replace("</p>", EMPTY).replace(NEW_LINE, EMPTY)));
```

Comment:
I would replace a new line with a space, ideally, to have both elements separated. But maybe we always have a space?

Reply:
No, there is no way we could add it inside text. I just checked generated ASCII, HTML and did not mention there was no space between sentences (I see it now). Thank you, fixed.